lib/MPIPreferences/src/MPIPreferences.jl

```diff
@@ -77,11 +78,14 @@ function use_jll_binary(binary = Sys.iswindows() ? "MicrosoftMPI_jll" : "MPICH_j
               Accepted options are:
                   $(join(known_binaries, ", "))""")
     set_preferences!(MPIPreferences,
-        "_format" => "1.0",
+        "_format" => "1.1",
```

Comment:
```suggestion
        "_format" => "1.0",
```
since it should be backward compatible with older MPIPreferences

Reply:
Oh, so later versions can read earlier formats? Nice!